homework-selects/sql_questions.sql

```diff
@@ -4,54 +4,284 @@ SELECT first_name, last_name, salary, job_title
 FROM employees
          LEFT JOIN jobs USING (job_id);
 -- 2. the first and last name, department, city, and state province for each employee.
+SELECT FIRST_NAME, LAST_NAME, DEPARTMENT_NAME, STATE_PROVINCE
+FROM DEPARTMENTS
+         LEFT JOIN EMPLOYEES E on DEPARTMENTS.DEPARTMENT_ID = E.DEPARTMENT_ID
+         LEFT JOIN LOCATIONS L on DEPARTMENTS.LOCATION_ID = L.LOCATION_ID;
 -- 3. the first name, last name, department number and department name, for all employees for departments 80 or 40.
+SELECT FIRST_NAME, LAST_NAME, DEPARTMENT_ID, DEPARTMENT_NAME
+FROM EMPLOYEES
+         LEFT JOIN DEPARTMENTS
+                   USING (DEPARTMENT_ID)
+WHERE DEPARTMENT_ID = 40
+   or DEPARTMENT_ID = 80;
```

Comment:
please use SQL IN Operator for multiple OR conditions

ex: WHERE department_id IN (val1, val2, ...)

Reply:
fixed